README.md

```diff
@@ -5,6 +5,7 @@ export metrics over them.
 
 ## Getting started
 
+```
```

Comment:
This one actually breaks the flow

Reply:
What do you mean? Formatting?

Reply:
Yes, your added line needs to be removed again actually or the formatting is broken.

Reply:
OK. Thanks for pointing this out.